internal/keystore/keystore.go

```diff
@@ -42,7 +43,7 @@ func Unlock(account accounts.Account) {
 // SignTx signs transaction using account key
 func SignTx(account accounts.Account, tx *types.Transaction) (*types.Transaction, error) {
 	if hmyKeystore != nil {
-		return hmyKeystore.SignTx(account, tx, nil)
+		return hmyKeystore.SignTx(account, tx, big.NewInt(1)) // TODO: use chainID from config
```

Comment:
we need use chain config ID instead of 1 to distinguish mainnet from testnet and pangaea

Reply:
This needs wallet configuration. This part of code may not have reference to chainConfig.

Reply:
Done.